src/commands/__tests__/login.test.js

```diff
@@ -1,4 +1,30 @@
 // @flow
 import { login, toLoginOptions } from '../login';
+import * as npm from '../../utils/npm';
 
-test('bolt login');
+describe('bolt login', () => {
+  let npmLoginSpy;
+
+  beforeEach(() => {
+    npmLoginSpy = jest.spyOn(npm, 'login');
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
no need to clear mocks.
just mock npm module like
`jest.mock('../../utils/npm')` // just after imports
we are already have config to reset mocks in jest.config.js in root

Reply:
sorry it is package json - https://github.com/boltpkg/bolt/blob/master/package.json#L76

Reply:
Updated!!